lighthouse-core/lib/page-functions.js

```diff
@@ -27,7 +27,6 @@
  * promise. Instead, map to a successful object that contains this information.
  * @param {string|Error} err The error to convert
  */
-/* istanbul ignore next */
```

Comment:
maybe worth adding to the fileoverview that this whole file is ignored for code coverage since it's intended to run in the browser?

Reply:
Is that information useful to someone looking at this file? Seems a concern only if you are looking at how c8 works, and naturally you'd be looking at c8.sh.

Reply:
> Is that information useful to someone looking/editing/using functions form this module?

I think so. There's coverage magic going on and the only way to figure out why e.g. a new page function in here doesn't have to be ignored while a new page function that's defined within a gatherer file does have to be ignored is to know you have to look in `c8.sh` in the first place